packages/access/lib/accessScheduler.js

```diff
@@ -29,7 +30,8 @@ const init = async ({ mail }) => {
         await schedulerLock()
           .lock('locks:access-scheduler', 1000 * intervalSecs)
```

Comment:
should the TTL not decrease here? Am I right that you lock here for all of the interval, just to be sure?

Reply:
Not sure why it should be decreased.

It does indeed lock the entire interval. If it succeeds, it will [extend lock by a 10th](https://github.com/orbiting/backends/pull/107/files/0488617e44f33bf09ba9506d8f6301cbc100b92f#diff-41d11ac02aa1af5acae07bf411f9bfccR38) to prevent back-to-back runs (and somewhat decrease it).